src/stories/pages/AddExperiment/AddExperiment.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import ExperimentServices
+  from '../../../firebase/CRUDServices/experimentServices';
+
+import { FormItem } from '../../components/FormItem/FormItem';
+import { Form } from '../../components/Form/Form';
+import { Button } from '../../components/Button/Button';
+import { Modal } from '../../components/Modal/Modal';
+
+/**
+ * Component for AddExperiment page.
+ * @param {string} theme theme of the parent component.
+ * @param {string} buttonText text to appear on the open button.
+ * @component
+ * @return {object} (
+ *   <AddExperiment isDarkTheme={isDarkTheme}>
+ * )
+ */
+
+export const AddExperiment = ({ theme, buttonText }) => {
+  const [error, setError] = useState('');
+  const [title, setTitle] = useState('');
+  const [shortDesc, setShortDesc] = useState('');
+  const [longDesc, setLongDesc] = useState('');
+  const [qualtrics, setQualtrics] = useState('');
+  const [spych, setSpych] = useState('');
```

Comment:
Does this refer to jsPsych? If it does maybe we should change this to `psych` and `setPsych`?